src/Cafe/HW/Latte/Renderer/Vulkan/TextureReadbackVk.cpp

```diff
@@ -79,6 +79,16 @@ uint32 LatteTextureReadbackInfoVk::GetImageSize(LatteTextureView* textureView)
 		// todo - if driver does not support VK_FORMAT_D24_UNORM_S8_UINT this is represented as VK_FORMAT_D32_SFLOAT_S8_UINT which is 8 bytes
 		return baseTexture->width * baseTexture->height * 4;
 	}
+	else if (textureView->format == Latte::E_GX2SURFFMT::R5_G6_B5_UNORM )
+	{
+		cemu_assert(textureFormat == VK_FORMAT_R8G8B8A8_UNORM);
+		return baseTexture->width * baseTexture->height * 4;
```

Comment:
```suggestion
		return baseTexture->width * baseTexture->height * 2;
```
`Latte::E_GX2SURFFMT::R5_G6_B5_UNORM` is 2 bytes per texel